dependencies.yaml

```diff
@@ -0,0 +1,101 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: conda
+    matrix:
+      cuda: ["11.5"]
+      arch: [x86_64]
+    includes:
+      - build
+      - cudatoolkit
+      - develop
+      - doc
+      - run
+      - test_python
+  test_python:
+    output: none
+    includes:
+      - cudatoolkit
+      - py_version
+      - test_python
+  test_cpp:
+    output: none
+    includes:
+      - cudatoolkit
+      - test_cpp 
+  checks:
+    output: none
+    includes:
+      - build
+      - develop
+      - py_version
+channels:
+  - rapidsai
+  - rapidsai-nightly
+  - conda-forge
+  - nvidia
+dependencies:
+  build:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - cudf=22.12.*
+          - cmake>=3.23.1
+          - cython>=0.29,<0.30
+          - python>=3.8,<3.10
+          - scikit-build>=0.13.1
+          - gdal>=3.0.2
+          - geopandas>=0.11.0
+  cudatoolkit:
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              cuda: "11.5"
+            packages:
+              - cudatoolkit=11.5
+  develop:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - pre-commit 
+      - output_types: conda
+        packages:
+          - clang=11.1.0
+          - clang-tools=11.1.0
+  doc:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - pydata-sphinx-theme
+          - myst-parser
+  py_version:
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              py: "3.8"
+            packages:
+              - python=3.8
+          - matrix:
+              py: "3.9"
+            packages:
+              - python=3.9
+  run:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - numba>=0.49
```

Comment:
Have we reviewed these >= values recently? These versions of Numba and NumPy are getting kind of old. Do the minimum versions still work?

Reply:
Cuspatial doesn't really depend on the use of numba as I don't think it hosts any jitted kernel. Should remove probably.

Reply:
Removed